websites/Z/Zoom/presence.ts

```diff
@@ -66,3 +81,14 @@ function videoEnabled() {
       .getAttribute("aria-label") != "start sending my video"
   );
 }
+
+function memberCount() {
+  const counter = document.querySelector(
+    ".footer-button__participants-icon > .footer-button__number-counter > span"
+  );
+  if (counter == null) {
```

Comment:
```suggestion
  if (counter === null) {
```

Reply:
Don't think that's actually a good idea here, if for some reason a browser return `undefined` from `document.querySelector`  it will fail